source/nodejs/adaptivecards/src/carousel.ts

```diff
@@ -116,8 +117,17 @@ export class Carousel extends Container {
 
     //#endregion
 
+    get currentEventType(): Enums.CarouselEventType {
+        return this._currentEventType;
```

Comment:
what's this._currentEventType for?

Reply:
I expecting it's fire and forget

Reply:
swiper doesn't have pagenation event. so we listen for two events, events that cause the slide transition and end of slide transition event. we reset the state at the end of slide transition.